frontend/src/views/main/project/InspectorWrapper.vue

```diff
@@ -434,6 +455,26 @@ async function handleSwitchSample() {
     }
   });
 }
+
+const modelFeatures = computed(() => {
```

Comment:
I think this can be removed totally since it just return a copy of `inputMetaData`